### DISCUSSION TITLE
Slingbox device adminpassword recovery

### DISCUSSION THREAD

**Discussion by dgtl** [GerryDazoo/Slinger#521] [General]
I'm kinda at a loss, with trying the recover my sling device admin password.

I see the readme but it's gibberish to me, any assistance is appreaciated.

**Comment by swilsonnc** [comment]
A little more info might help.  Which device?  Have you factory reset it?  Have you tried admin or password for the passwords?

**Comment by dgtl** [comment]
sorry for the delay, 
I did try "admin" & "password. no luck. I'm trying to recover password for a 500 located at '47.149.132.143', 5216 .

**Reply by GerryDazoo** [reply]
you can't recover the password from the box. What happened when you tried the password recovery procedure?
https://github.com/GerryDazoo/Slinger/discussions/389
